spk/rutorrent/src/service-setup.sh

```diff
@@ -0,0 +1,289 @@
+    fi
+}
+
+service_preinst ()
+{
+    if [ "${SYNOPKG_PKG_STATUS}" == "INSTALL" ]; then
+        if [ ! -d "${wizard_download_dir}" ]; then
+            echo "Download directory ${wizard_download_dir} does not exist."
+            exit 1
+        fi
+        if [ -n "${wizard_watch_dir}" -a ! -d "${wizard_watch_dir}" ]; then
+            echo "Watch directory ${wizard_watch_dir} does not exist."
+            exit 1
+        fi
+    fi
+
+    exit 0
+}
+
+fix_shared_folders_rights()
+{
+    local folder=$1
+    echo "Fixing shared folder rights for ${folder}" >> "${INST_LOG}"
+    chown -R "${APACHE_USER}:${USER}" "${folder}" >> "${INST_LOG}" 2>&1
+    chmod ug+rwx "${folder}" >> "${INST_LOG}" 2>&1
+    synoacltool -add "${folder}" "user:${EFF_USER}:allow:rwxpdDaARWcC:fd" >> "${INST_LOG}" 2>&1
+    synoacltool -add "${folder}" "user:${APACHE_USER}:allow:rwxpdDaARWcC:fd" >> "${INST_LOG}" 2>&1
+    find "${folder}" -type d -mindepth 1 -exec synoacltool -enforce-inherit "{}" \; >> ${INST_LOG} 2>&1
+}
+
+service_postinst ()
+{
+
+    # Install busybox stuff
+    ${INSTALL_DIR}/bin/busybox --install ${INSTALL_DIR}/bin
+
+    syno_user_add_to_legacy_group "${EFF_USER}" "${LEGACY_USER}" "${LEGACY_GROUP}"
+
+    # Install the web interface
+    cp -pR ${INSTALL_DIR}/share/${PACKAGE} ${WEB_DIR} >>"${INST_LOG}" 2>&1
+
+    # Configure open_basedir
+    if [ "${APACHE_USER}" == "nobody" ]; then
+        echo -e "<Directory \"${WEB_DIR}/${PACKAGE}\">\nphp_admin_value open_basedir none\n</Directory>" > /usr/syno/etc/sites-enabled-user/${PACKAGE}.conf
+    else
+        if [ -d "/etc/php/conf.d/" ]; then
+            echo -e "[PATH=${WEB_DIR}/${PACKAGE}]\nopen_basedir = Null" > /etc/php/conf.d/${PACKAGE_NAME}.ini
+        fi
+    fi
+
+    # Configure files
+    if [ "${SYNOPKG_PKG_STATUS}" == "INSTALL" ]; then
+        TOP_DIR=`echo "${wizard_download_dir:=/volume1/downloads}" | cut -d "/" -f 2`
+        MAX_MEMORY=`awk '/MemTotal/{memory=$2*1024*0.25; if (memory > 512*1024*1024) memory=512*1024*1024; printf "%0.f", memory}' /proc/meminfo`
+
+        sed -i -e "s|scgi_port = 5000;|scgi_port = 8050;|g" \
+               -e "s|topDirectory = '/';|topDirectory = '/${TOP_DIR}/';|g" \
+               -e "s|tempDirectory = null;|tempDirectory = '${INSTALL_DIR}/tmp';|g" \
+               -e "s|\"python\"\(\\s*\)=>\(\\s*\)'.*'\(\\s*\),\(\\s*\)|\"python\"\1=>\2'${INSTALL_DIR}/env/bin/python3'\3,\4|g" \
+               -e "s|\"pgrep\"\(\\s*\)=>\(\\s*\)'.*'\(\\s*\),\(\\s*\)|\"pgrep\"\1=>\2'${INSTALL_DIR}/bin/pgrep'\3,\4|g" \
+               -e "s|\"sox\"\(\\s*\)=>\(\\s*\)'.*'\(\\s*\),\(\\s*\)|\"sox\"\1=>\2'${INSTALL_DIR}/bin/sox'\3,\4|g" \
+               -e "s|\"mediainfo\"\(\\s*\)=>\(\\s*\)'.*'\(\\s*\),\(\\s*\)|\"mediainfo\"\1=>\2'${INSTALL_DIR}/bin/mediainfo'\3,\4|g" \
+               -e "s|\"stat\"\(\\s*\)=>\(\\s*\)'.*'\(\\s*\),\(\\s*\)|\"stat\"\1=>\2'/bin/stat'\3,\4|g" \
```

Comment:
I could replace the `/bin/stat` by a more prosaic `$(which stat)` but I guessed that would be a little bit far-fetched don't you think ?